web/src/components/Plugin/UI/limit-req.tsx

```diff
@@ -88,10 +95,21 @@ const LimitReq: React.FC<Props> = ({ form, schema }) => {
         initialValue={propertires.rejected_code.default}
         tooltip={formatMessage({ id: 'component.pluginForm.limit-req.rejected_code.tooltip' })}
       >
-        <InputNumber min={propertires.rejected_code.minimum} max={propertires.rejected_code.maximum} />
+        <InputNumber
+          min={propertires.rejected_code.minimum}
+          max={propertires.rejected_code.maximum}
+        />
+      </Form.Item>
+      <Form.Item
```

Comment:
@Baoyuantop I notice this PR's main change is here, and other changes are made by your IDE or some other tools? They confuse me when I review your PR, please don't change them in this PR.

BTW, please test this new field, you could ask @guoqqqi for help.

Reply:
when I `commit` code, the `lint-staged`  is executed, then the code becomes like this.

Does it seem that too much code was not executed by `lint-staged` before?🤔

Reply:
maybe, because `husky` is introduced days ago, and don't have a global lint.